ext_mongodb.php

```diff
@@ -21,6 +21,11 @@ public function getMessage() : string
 		return $this->message;
 	}
 
+	public function getInfo() : ?array
```

Comment:
@derickr: Per [this documentation](http://docs.hhvm.com/manual/en/hack.nullable.scenarios.php#hack.nullable.scenarios.nonnullableinteraction)), we can use nullables for return types.